doc/guides/writing_tests.md

```diff
@@ -78,10 +78,9 @@ This is the body of the test. This test is quite simple, it just tests that an
 HTTP server accepts `non-ASCII` characters in the headers of an incoming
 request. Interesting things to notice:
 
-- The use of `common.PORT` as the listening port. Always use `common.PORT`
-  instead of using an arbitrary value, as it allows to run tests in parallel
-  safely, as they are not trying to reuse the same port another test is already
-  using.
+- The use of 0 as the listening port. Always use 0 instead of using an
+  arbitrary value, as it allows to run tests in parallel safely, as the
+  operating system will assign a random port.
```

Comment:
We could add something explaining that this is true, unless the test is specifically checking that assigning a specific port works as expected.